docs/source/config_overview.md

```diff
@@ -26,6 +26,10 @@ and editing settings is similar for all the Jupyter applications.
 > - [traitlets](https://traitlets.readthedocs.io/en/latest/config.html#module-traitlets.config)
 >   provide a low-level architecture for configuration.
 
+### Disabling Custom CSS
```

Comment:
As a follow-up (could be another PR), maybe it could be interesting to document the feature more (for example in the user guide), to show an example of such `custom.css` file.